Apps filter project categories

NielsKunst opened the pull request:
This is tied to a release of Sprintr. Release date is yet to be confirmed.

NielsKunst committed:
Projects Category documentation

NielsKunst committed:
Projects Category documentation

NielsKunst committed:
Add documentation for Project Categories

NielsKunst committed:
Add documentation for Project Categories

NielsKunst committed:
Merge branch 'apps-filter-project-categories' of https://github.com/NielsKunst/docs into apps-filter-project-categories

NicoletaComan committed:
TW review

NicoletaComan committed:
Updated per test env UI

NicoletaComan commented on content/en/docs/control-center/company/project-categories.md:
```diff
@@ -0,0 +1,45 @@
+2. In the **Category Name** field, enter a name for the new category.
+3. Click **Add Value** to add one or more values for the new category.   
+    You can add multiple values as a comma-separated list.    
+    Each entry consists of a unique name and code. If you add a new value without a code, the system automatically generates a unique code for you.      
```
@NielsKunst, I do not see the option to add a code for a value. Per my tests, the code is automatically assigned. Is it an error on my side or is that the way it's meant to work?

NielsKunst replied:
You are right, you can only edit the code after its creation. I have updated the wording.

NicoletaComan commented on content/en/docs/control-center/company/project-categories.md:
```diff
@@ -0,0 +1,45 @@
+
+## Category Management
+
+On the **Categories** page, you can manage app categories. A list of preconfigured categories is displayed, along with all their corresponding values. Next to the preconfigured **Country** category, you can configure a maximum of five categories with 250 values each. 
```
"A list of preconfigured categories is displayed, along with all their corresponding values."
Does this mean that **Country** won't be the only preconfigured category?

NielsKunst replied:
At this time, it is only the Country field. But we keep the option open to add more preconfigured categories in the future (if there is demand)

NielsKunst committed:
Update edit description of category value code

NicoletaComan committed:
Added release notes

NicoletaComan committed:
TW review

mxfmg1982 committed:
Correct release date and enhance Project Categories details

Updated the release notes for December 2025 to reflect the correct date and expanded the description of the new Project Categories feature in the Control Center.

mxfmg1982 committed:
Change release date from December 15 to 14

Updated the release date for December 2025.

NicoletaComan committed:
Updated latest releases & layout

NicoletaComan committed:
Merge branch 'development' into apps-filter-project-categories
